assets/styl/_base/symbols.json

```diff
@@ -0,0 +1,28 @@
+{
+    "symbols": [
+        "clock",
+        "comments",
+        "envelope",
+        "home",
+        "logo",
+        "menu",
+        "newspaper",
+        "plus",
+        "search",
+        "settings",
+        "switch",
+        "user",
+        "arrow-up",
+        "arrow-down",
+        "arrow-left",
+        "arrow-right",
+        "social-facebook",
+        "social-github",
+        "social-googleplus",
+        "social-linkedin",
+        "social-pinterest",
+        "social-snapchat",
+        "social-twitter",
+        "social-whatsapp"
```

Comment:
I would remove the `social-` prefixe. As font and svg icons are going to coexist for a while, I think we should keep icon namings consistent: we should keep the same aliases defined in [_helpers/_icons.styl](https://github.com/20minutes/colette/blob/master/assets/styl/_helpers/_icons.styl), and even use them for svg file names.

This can be discussed.